keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -205,14 +244,19 @@ restoring from various kinds of things.
 
 ### Implementation Details/Notes/Constraints
 
-As noted above, the proposal is extremely simple -- just remove the validation on
-the `DataSource` field. This raises the question of WHAT will happen when users
+As noted above, the proposal recommends a modest change to the PVC API which
+creates a new more flexible field to specify data sources without breaking any
+existing behaviors. Mostly the new `DataSourceRef` will never ignore user input,
+so that arbitrary CRDs can be used as data sources.
+This raises the question of WHAT will happen when users
 put new things in that field, and HOW populators will actually work with so small
 a change.
 
 It's first important to note that only consumers of the `DataSource` field today
 are the various dynamic provisioners, most notably the external-provisioner CSI
-sidecar. If the external-provisioner sidecar sees a data source it doesn't
+sidecar. Due to the API change, these consumers will need to switch to consume the
```

Comment:
Actually - consumers don't NEED to switch, since we arranged that `old` is set from `new`

Reply:
Right, but if we carry through on deprecation and only documenting the new field, then they SHOULD switch even if it's not a MUST that they switch. I wrote this before the latest update, and I'm happy to amend it.

Reply:
I just had to find SOMETHING to pick at :)